File: lecture_002/N_numbers.java
import java.util.Scanner;

public class N_numbers{
	public static void main(String[] args){
		Scanner scn=new Scanner(System.in);
		System.out.print("enter the Nth number: ");
		int n = scn.nextInt();
		//  for loop 
    	for(int i = 1; i <= n; i++){            //  here int i has scope upto end of the loop, not beyond that
        	System.out.println(i);
    	}
    	//  end of for loop
	}
}

File: lecture_002/N_numbers.cpp
//  1 to N numbers

#include<iostream>
using namespace std;
int main(int args, char**argv)
{
    int n;
    cout<<"enter the nth number: ";
    cin>>n;

    //  for loop 
    for(int i = 1; i <= n; i++){            //  here int i has scope upto end of the loop, not beyond that
        cout<<i<<endl;
    }
    //  end of for loop

    return 0;
}